CHANGELOG.react.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   Fix stroke-width property on react progress bar.
 -   Fixes to Text Area related to Input resizing and `valid` and `error` states in Material
+-   Text field enhancements (`before`, `after`, `onFocus`, `onBlur`, `textfieldRef`, `inputRef` props) [#199](https://github.com/lumapps/design-system/pull/199)
```

Comment:
It could be under `Unreleased > Added` rather than `Changed`, what do you think?

Reply:
Also I don't see the `before` and `after` props in `ITextFieldProps`